lib/view/templates.py

```diff
@@ -1458,8 +1560,8 @@ def numeric_sum_aggregator_selector(key, is_numeric):
 
 
 def latency_weighted_avg(edatas: list[EntryData]):
-    pcts = map(lambda edata: edata.value, edatas)
-    ops_sec_values = map(lambda edata: edata.record["ops/sec"], edatas)
+    ops_sec_values: map[float] = map(lambda edata: edata.record["ops/sec"], edatas)
```

Comment:
is this a valid type hint? isn't the result of map() a list?

Reply:
It is valid. A map returns a generator like object called a map